packages/dev/core/src/ShadersWGSL/geometry.vertex.fx

```diff
@@ -155,7 +158,7 @@ fn main(input : VertexInputs) -> FragmentInputs {
 			#if defined(ALPHATEST) && defined(ALPHATEST_UV1)
 			vertexOutputs.vUV = (uniforms.diffuseMatrix *  vec4f(uvUpdated, 1.0, 0.0)).xy;
 			#else
-			vertexOutputs.vUV = input.uv;
+			vertexOutputs.vUV = uvUpdated;

```

Comment:
I'm not sure about this. Looks like we should use `uvUpdated` instead of `input.uv`?

Reply:
Yes, good catch! Can you also fix the same bug in the WebGL code of `geometry.vertex.fx`? Thanks!